pkg/controllers/kepler_internal.go

```diff
@@ -199,7 +200,7 @@ func (r KeplerInternalReconciler) updateReconciledStatus(ctx context.Context, ki
 func (r KeplerInternalReconciler) updateAvailableStatus(ctx context.Context, ki *v1alpha1.KeplerInternal, recErr error) {
 	// get daemonset owned by kepler
 	dset := appsv1.DaemonSet{}
-	key := types.NamespacedName{Name: ki.DaemonsetName(), Namespace: ki.Namespace()}
+	key := types.NamespacedName{Name: ki.DaemonsetName(), Namespace: ki.Spec.Exporter.Deployment.Namespace}
```

Comment:
@sunya-ch , How about we move `spec.exporter.deployment.namespace` to `spec.namespace` so that all components of `ki` will use this common field for deployment and can't be empty

Reply:
First, change ki.Namespace to ki.Spec.Exporter.Deployment.Namespace is another bug I fix.
ki (keplerinternal CR) is cluster-wide. ki.Namespace will give an empty "" namespace. I don't know when this bug occur but it made the status failed to update.

Second, I do agree to move the namespace from exporter.deployment.namespace to outside. But, in that case, we actually show move node-selector and toleration to the same level since it shares between estimator and exporter.  I also notice that it is missing in the internal CR. 
We can keep as is and have the estimator and model server uses it as a reference. Or let the model server has its own namespace. WDYT?

Reply:
`ki.Namespace()` must never return an empty string since kubernetes API validation should prevent .t
could you please revert this change?

Reply:
> Second, I do agree to move the namespace from exporter.deployment.namespace to outside. But, in that case, we actually show move node-selector and toleration to the same level since it shares between estimator and exporter. 

A seperate Node selector for kepler makes-sense but not for model-server. The use-case for kepler is that it deploys a daemonset and pods are deployed per node. so giving the ability to decide which nodes to monitor is a must have and should be separate to deploying a single instance of model-server in *any* node, shouldn't it?

Reply:
Yes, model server is separated but estimator shared the same. What I think is maybe we can keep the .deployment in exporter spec with namespace, selector, toleration. 
Meanwhile, model-sever may also have its own deployment spec which have a namespace.

Reply:
> `ki.Namespace()` must never return an empty string since kubernetes API validation should prevent .t
> could you please revert this change? If you experience an issue with it, lets fix that separately.

I just noticed it is the function call. I think I mixed the issue. Now revert back.